### PR TITLE
Team1 toi dt module latest car new popular new cars new makes

### PR DESCRIPTION
**Pull request by DoThiToi**
Verify that module LatestCarNews displayed list six article and link
Verify that link 'See All Latest News' in module Latest Car News should be clickable
Verify that content 'About the Kelley Blue Book New Car Reviews Page' should be displayed with the short description about it
Verify that module search Shop for Top Rated Cars should be displayed twelve item
Verify that brand name in module search Shop for Top Rated Cars should be clickable
Verify that module search Popular New Makes should be displayed twenty-four itemV
Verify that brand name in module search 'Popular New Makes' should be clickable.

### CODE REVIEW

**Comment by phkhuong2** [comment]
10 điểm

**Review by phkhuong2 — APPROVED** [review]
good good

File: demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/AboutTheKelleyBlueBookNewCarReviewsPage.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+public class AboutTheKelleyBlueBookNewCarReviewsPage extends BasePage {
+
+    public AboutTheKelleyBlueBookNewCarReviewsPage(WebDriver driver) {
+        super(driver);
+    }
+
+    public AboutTheKelleyBlueBookNewCarReviewsPage() {
+    }
+    
+    //
```
Comment (ntmtrang) [inline demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/AboutTheKelleyBlueBookNewCarReviewsPage.java]: If you don't use these code, you should delete it, don't comment like that

File: demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/ShopForTopRatedCarsPage.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    // private By txtHondaCR = By.xpath("//*[@id='strc-honda-cr-v-5']");
+    // private By txtHondaCiviv = By.xpath("//*[@id='strc-honda-civic-9']");
+    // private By txtNissan = By.xpath("//*[@id='strc-nissan-rogue-2']");
+    // private By txtToyotaCorolla = By.xpath("//*[@id='strc-toyota-corolla-6']");
+    // private By txtHondaAccord = By.xpath("//*[@id='strc-honda-accord-10']");
+    // private By txtJeep = By.xpath("//*[@id='strc-jeep-wrangler-3']");
+    // private By txtFordRanger = By.xpath("//*[@id='strc-ford-ranger-7']");
+    // private By txtSubaru = By.xpath("//*[@id='strc-subaru-outback-11']");
+
+//
```
Comment (ntmtrang) [inline demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/ShopForTopRatedCarsPage.java]: If you don't use these code, you should delete it, don't comment like that

File: demo-automation-java/src/test/java/com/example/tests/car_reviews_tests/CarReviewsLatestCarNewTest.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+import com.example.pages.car_reviews_pages.PopularNewMakesPage;
+import com.example.pages.car_reviews_pages.ShopForTopRatedCarsPage;
+import com.example.tests.BaseTest;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CarReviewsLatestCarNewTest extends BaseTest {
+    
+    // @Test(testName = "Demo Navigate To The Page")
```
Comment (ntmtrang) [inline demo-automation-java/src/test/java/com/example/tests/car_reviews_tests/CarReviewsLatestCarNewTest.java]: If you don't use these code, you should delete it, don't comment like that

**Review by ntmtrang — DISMISSED** [review]

File: demo-automation-java/src/test/java/com/example/tests/car_reviews_tests/CarReviewsLatestCarNewTest.java (hunk 3 above)
Comment (DoThiToi) [reply demo-automation-java/src/test/java/com/example/tests/car_reviews_tests/CarReviewsLatestCarNewTest.java]: Thank you for reviewed. I will update

**Pull request merged**